modules/hostap/src/supp_main.c

```diff
@@ -306,9 +306,32 @@ static void iface_event_handler(struct net_mgmt_event_callback *cb,
 							uint32_t mgmt_event, struct net_if *iface)
 {
 	const char *ifname = iface->if_dev->dev->name;
+	char *tmp_csv;
+	char *prefix;
+	char *mgd_if_prefix = CONFIG_WPA_SUPP_MGD_IFACES_PREFIX;
```

Comment:
We can do many things in built time with following changes.
Instead of using a pointer, we could do this (as the string is small the impact to stack is negligible)
```
char mgd_if_prefix[] = CONFIG_WPA_SUPP_MGD_IFACES_PREFIX;
```

Reply:
Done.